locales/en/messages.json

```diff
@@ -1054,7 +1054,7 @@
         "message": "In-flight level calibration"
     },
     "featureRX_SERIAL": {
-        "message": "Serial-based receiver (SPEKSAT, SBUS, SUMD)"
+        "message": "Serial (CRSF, SBUS, Ghost...)"
```

Comment:
No issue with adding more proprietary protocols, but I think we should leave the open source alternatives like  SUMD in the list.

Reply:
Do we even need to list any protocols here? All of them will appear in the provider dropdown when serial is selected.

Reply:
The only reason they are there is as an example, to help the noob decide which to choose.  
We don't really **need** anything other than the protocol name imho.  
However perhaps for beginners it is easier to give a kind of pseudo-explanation.

Reply:
I would agree here with @ctzsnooze - it's for most people to make the dropdown choice easier when they set up drones.